templates/support_channels.html

```diff
@@ -0,0 +1,94 @@
+{% load i18n %}
+{% load static %}
+
+<section class="frontpage frontpage-section">
+	<div class="cards">
+		<div class="card frontpage">
+			<div class="card-body">
+				<h3>{% trans "Documentation" %}</h3>
+				<p class="card-text">
+					<a target="_blank" rel="noopener"
+						href="https://plus.cs.aalto.fi/aplus-manual/master/">{% trans "Aplus manual" %}</a>
+				</p>
+				<p class="card-text">
+					<a target="_blank" rel="noopener" href="https://apluslms.github.io/">{% trans "Aplus website" %}</a>
+				</p>
+			</div>
+		</div>
+
+		<div class="card frontpage">
+			<div class="card-body">
+				<h3>{% trans "Community" %}</h3>
+				<p class="card-text">
+					<a target="_blank" rel="noopener"
+						href="https://teams.microsoft.com/l/team/19%3a147685e6a8fc409295188befeec77113%40thread.skype/conversations?groupId=b78ab778-8838-4180-940d-b9183ddc3a7c&tenantId=ae1a7724-4041-4462-a6dc-538cb199707e">
+						{% trans "Microsoft Teams" %}
+					</a>
+				</p>
+				<p class="card-text">
+					<a target="_blank" rel="noopener"
+						href="https://github.com/apluslms/a-plus/issues">{% trans "Create an issue in Github" %}</a>
+				</p>
+			</div>
+		</div>
+
+		<div class="card frontpage">
+			<div class="card-body">
+				<h3>{% trans "News" %}</h3>
+				<p class="card-text">
+					<a target="_blank" rel="noopener"
+						href="https://wiki.aalto.fi/display/EDIT/EDIT#EDIT-News">{% trans "Announcements" %}</a>
+				</p>
+				<p class="card-text">
+					<a target="_blank" rel="noopener"
+						href="https://status.cs.aalto.fi/d/r0N6S9jik/main?orgId=1">{% trans "Status" %}</a>
+				</p>
+				<p class="card-text">
+					<a target="_blank" rel="noopener"
+						href="https://github.com/orgs/apluslms/projects/2#card-50639420">{% trans "Current Development" %}</a>
+				</p>
+			</div>
+		</div>
+
+		<div class="card frontpage">
+			<div class="card-body">
+				<h3>{% trans "Other servers that use A+" %}</h3>
+				<p class="card-text">
+					<a target="_blank" rel="noopener" href="https://tentit.cs.aalto.fi/">{% trans "Exam server" %}</a>
```

Comment:
Our tentit.cs server is protected by a firewall, so it is not useful to link to it. However, our A+ EXAM instructions in the wiki would be useful: https://wiki.aalto.fi/display/EDIT/Exam+studio+exam+with+Aplus+exercises

You already included this link in the next section, though.

Reply:
But some teachers have access to the tentit server, don't they?

Reply:
Yes, those teachers who have requested access. The wiki contains instructions for how to request access.